podcasts/Onboarding/LoginCoordinator.swift

```diff
@@ -154,6 +154,12 @@ extension LoginCoordinator: SyncSigninDelegate, CreateAccountDelegate {
      }
 
     func signingProcessCompleted() {
+        // Due to connection issues this might be called even if the user didn't actually
+        // signed in. So we make sure the user is actually logged in.
+        guard SyncManager.isUserLoggedIn() else {
```

Comment:
Another approach would be to change the notification to not be triggered, but... we don't know what this might cause, so safer to go with this.